surveys/2021-annual-survey/questions.md

```diff
@@ -716,15 +716,15 @@ Select one:
 Select one:
 
 - Not enough usage in industry
-- Too much interest from industry leads to too much politics
-- Not enough users
-- Doesn't add a feature I want
-- Stagnation
+- Too much interest from big companies
+- Not enough open source contributions to the ecosystem
+- Doesn't add a specific feature I want
+- Rust doesn't evolve quickly enough
 - Instability of the language
 - Superseded by an alternative
 - Becomes too complex
-- Not enough localization
-- Other (open response)
+- Tools and documentation are not accessible enough (e.g., due to language or incompatibility with screen readers)
+- Other
```

Comment:
Do we not want to allow for open responses?

Reply:
Hmm I'm not sure that's true. Either way, we have lots of other "Other (open response)" answers so I'd prefer to tackle them all at once if possible.